src/components/dropzone/dropzone.tsx

```diff
@@ -110,6 +110,13 @@ const Dropzone = () => {
     }
   }, [globalReset])
 
+  useEffect(() => {
+    setDropzoneRef(dropzoneRef)
+    return () => {
+      delete FileStore.dropzoneRef
+    }
+  }, [])
+
```

Comment:
Now dropzone ref can be accessed globally and opened with `FileStore.openFileDrawer`